src/index.scss

```diff
@@ -0,0 +1 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
```

Comment:
The index.css file should have been renamed to index.scss. See https://www.carbondesignsystem.com/tutorial/react/step-1/#add-ui-shell-sass

Also you're missing the UI shell import. https://www.carbondesignsystem.com/tutorial/react/step-1/#add-ui-shell-sass